packages/api-client-core/src/operationBuilders.ts

```diff
@@ -5,10 +5,14 @@ import type { AnyActionFunction } from "./index.js";
 import { camelize, capitalizeIdentifier, filterTypeName, sortTypeName } from "./support.js";
 import type { ActionFunctionOptions, BaseFindOptions, EnqueueBackgroundActionOptions, FindManyOptions, VariablesOptions } from "./types.js";
 
-const hydrationOptions = (modelApiIdentifier: string): BuilderFieldSelection => {
+const hydrationOptions = (modelApiIdentifier: string, namespace?: string | string[] | null): BuilderFieldSelection => {
+  const fullyQualifiedIdentifier = namespace
+    ? [...(Array.isArray(namespace) ? namespace : [namespace]), modelApiIdentifier].join(".")
+    : modelApiIdentifier;
+
   return {
     gadgetMeta: {
-      [`hydrations(modelName: "${modelApiIdentifier}")`]: true,
+      [`hydrations(modelName: "${fullyQualifiedIdentifier}")`]: true,
```

Comment:
Have we handled the dot notation style of "namespace + API identifier" on the Gadget side?